lib/config/rules/react.js

```diff
@@ -1,8 +1,12 @@
 module.exports = {
   // General
 
+  // Enforces consistent naming for boolean props <FEEDBACK>
+  'react/boolean-prop-naming': 'off',
```

Comment:
This seems useful but could cause some frustrations adopting in a large code base. Doesn't seem to have a fixer either.

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/boolean-prop-naming.md

Reply:
It would be cool if we could do the opposite of the default (never start with is/ has), as this is our standard.

Reply:
What the reasoning behind that?

Reply:
🤷‍♂️ it's just kind of what we landed on, it's consistent with our Ruby components and it avoids some kind of unnecessary duplication from my perspective